cmd/pipeline/main.go

```diff
@@ -437,6 +440,18 @@ func main() {
 			statusChangeDurationMetric,
 			clusterTotalMetric,
 		),
+		PKEOnVsphere: vspherePKEDriver.MakeVspherePKEClusterCreator(
+			vspherePKEDriver.ClusterCreatorConfig{
+				OIDCIssuerURL:               config.Auth.OIDC.Issuer,
+				PipelineExternalURL:         externalBaseURL,
+				PipelineExternalURLInsecure: externalURLInsecure,
+			},
+			logrusLogger,
```

Comment:
Please don't use the logrus logger, it's there for legacy reasons. 

Please use the common logger (and alias it in the necessary packages):

- https://github.com/banzaicloud/pipeline/blob/master/internal/common/logger.go#L22
- https://github.com/banzaicloud/pipeline/blob/master/internal/app/frontend/notification/common.go#L25